hudi-spark-datasource/hudi-spark/src/test/scala/org/apache/spark/sql/hudi/TestInsertTable.scala

```diff
@@ -1499,4 +1499,63 @@ class TestInsertTable extends HoodieSparkSqlTestBase {
       assertEquals(3, df.select(HoodieRecord.RECORD_KEY_METADATA_FIELD).count())
     }
   }
+
+  test("Test Bulk Insert Into Consistent Hashing Bucket Index Table") {
+    withSQLConf("hoodie.datasource.write.operation" -> "bulk_insert") {
+      withTempDir { tmp =>
+        val tableName = generateTableName
+        // Create a partitioned table
```

Comment:
does this mean that the consistent hashing index does not have the row writer code path support yet?

Reply:
Yes. And I would like to contribute this part if possible. However in 0.14 release, we should support regular bulk insert by default.